### PR TITLE
enable testDDRExtJunit_StackMap in DDR

### PR DESCRIPTION
**Pull request by OscarQQ**
- merge openj9 and ibm into one test
- use COREGEN in tck_ddrext.xml to specify which core generator to use

related issue: #1511

Signed-off-by: Yixin Qian <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
@llxia

File: test/functional/DDR_Test/playlist.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<include>ddrSettings.mk</include>
 	<test>
 		<testCaseName>testDDRExt_General_ibm</testCaseName>
-		<command>ant -DJAVA_COMMAND=$(JAVA_COMMAND) -DTEST_ROOT=${TEST_ROOT} -DTEST_JDK_HOME=${TEST_JDK_HOME} -DJDK_VERSION=${JDK_VERSION} \
+		<command>ant -DJAVA_COMMAND=$(JAVA_COMMAND) -DCOREGEN=$(Q)j9vm.test.corehelper.CoreGen$(Q) -DTEST_ROOT=${TEST_ROOT} -DTEST_JDK_HOME=${TEST_JDK_HOME} -DJDK_VERSION=${JDK_VERSION} \
```
Comment (llxia) [inline test/functional/DDR_Test/playlist.xml:27]: Please use a default value in xml, so we just need to set it for StackMap

File: test/functional/DDR_Test/playlist.xml [hunk 2]
```diff
@@ -202,4 +202,40 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
+		<!-- temporarily disable this test on z/OS; github.com/eclipse/openj9/issues/1511 -->
+		<platformRequirements>^os.zos</platformRequirements>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<impls>
+			<impl>openj9</impl>
```
Comment (llxia) [inline test/functional/DDR_Test/playlist.xml:237]: as mentioned last time, we can merge _ibm and _openj9 into one. We do not need to exclude on zos.

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by llxia** [comment]
[TestSoftMxCallBackExtend_1](https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_s390x_linux_Personal_testList_0/13/tapResults/) failure is related to https://github.com/eclipse/openj9/issues/10189

**Comment by OscarQQ** [comment]
| Build       | Platform           | Java Version  | JDK Impl |
|:-------------:|:-------------:|:-----:|:-----------:|
| Grinder/9075     | ppc64le_linux | 8 | openj9|
| Grinder/9077     | s390x_linux      |   8 | openj9|
| Grinder/9078 | x86-64_linux      |    8 | openj9|
| Grinder/9079 | x86-64_windows      |    8 | openj9|
| Grinder/9080 | x86-64_mac      |    8 | openj9|
| Grinder/9081 | ppc64_aix     |    8 | openj9|
| Grinder/9095     | ppc64le_linux | 8 | IBM|
| Grinder/9096     | s390x_linux      |   8 | IBM|
| Grinder/9097 | x86-64_linux      |    8 | IBM|
| Grinder/9098 | x86-64_windows      |    8 | IBM|
| Grinder/9099 | s390x_zos     |    8 | IBM|
| Grinder/9100 | ppc64_aix     |    8 | IBM|

**Pull request merged**